src/python/pants/notes/master.rst

```diff
@@ -4,6 +4,141 @@ Master Pre-Releases
 This document describes development releases which occur weekly from master, and which have
 not undergone the vetting associated with ``stable`` releases.
 
+1.25.0.dev3 (01/29/2020)
+------------------------
+
+API Changes
+~~~~~~~~~~~
+
+* Allow excludes in BUILD files via new `!` syntax (#9025)
+  `PR #9025 <https://github.com/pantsbuild/pants/pull/9025>`_
+
+* Remove the generate-pants-ini goal. (#8993)
+  `PR #8993 <https://github.com/pantsbuild/pants/pull/8993>`_
+
+* Deprecate `--fast` option (#8970)
+  `PR #8970 <https://github.com/pantsbuild/pants/pull/8970>`_
+
+* Switch to dot idea project type (#8745)
+  `PR #8745 <https://github.com/pantsbuild/pants/pull/8745>`_
+
+* Deprecate defaulting to `--transitive` (#8955)
+  `PR #8955 <https://github.com/pantsbuild/pants/pull/8955>`_
+
+* Add IPython subsystem and deprecate `--repl-py` options (#8996)
+  `PR #8996 <https://github.com/pantsbuild/pants/pull/8996>`_
+
+* Register certain optional conditionally on v1/v2 setup. (#8997)
+  `PR #8997 <https://github.com/pantsbuild/pants/pull/8997>`_
+
+New Features
+~~~~~~~~~~~~
+
+* Allow filesystem specs with `cloc2` and `validate` (#8985)
+  `PR #8985 <https://github.com/pantsbuild/pants/pull/8985>`_
+
+* Allow for deprecation of child scopes (#8984)
+  `PR #8984 <https://github.com/pantsbuild/pants/pull/8984>`_
```

Comment:
Hm, realized this is a refactoring. This only impacts us Pants devs. Even plugin authors likely aren't making deprecations.